new/PRS14025Balca.xml

```diff
@@ -1,71 +0,0 @@
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
```

Comment:
i believe this new record has not been yet merged and thus MAY NOT be deleted.
if you create a record, even erroneously, it first must be merged, then you update your master, and then you create a NEW clean PR from master for deletion. Otherwise the record will remain in the app.

when you create a record through the app betamasaheft.eu it is saved in the app. when you download it you have it then in your own branch. only when merged it is then saved in master branch - which must always be in sync with the app. then if you delete it from master it will also get deleted in the app. if it never makes it to master on github it will always remain on the app as a doublet (in the original stub form).

Reply:
Now I see - you did not even delete the file but deleted its content. This is something you may not do EVER as it renders the XML invalid. It may not be empty. Merge it WITH CONTENT. And then DELETE THE FILE not the content.

Reply:
OK.